drivers/src/lms.rs

```diff
@@ -301,6 +301,68 @@ impl Lms {
         Ok(HashValue::from(digest))
     }
 
+    #[cfg(feature = "hw-latest")]
+    pub fn hash_chain<const N: usize>(
+        &self,
+        sha256_driver: &mut impl Sha256Alg,
+        wnt_prefix: &mut [u8; 55],
+        coeff: u8,
+        params: &LmotsParameter,
+        tmp: &mut HashValue<N>,
+    ) -> CaliptraResult<HashValue<N>> {
+        const WNTZ_MODE_SHA256: u8 = 32;
+
+        let iteration_count = ((1u16 << params.w) - 1) as u8;
+
+        if coeff < iteration_count {
+            let mut digest = Array4x8::default();
+            let mut hasher = sha256_driver.digest_init()?;
+            wnt_prefix[22] = coeff;
```

Comment:
There are a lot of hard-coded values in this function. Can we declare constants for these so the program logic is clearer to the reader?

Reply:
Done.